test/Tests/src/Semantic/Meta_Spec.enso

```diff
@@ -258,10 +258,10 @@ spec =
             methods.sort . should_equal ['Value', 'create', 'factory', 'first_method', 'my_method', 'other_method', 'second_method']
 
         Test.specify "methods of Integer" <|
-            Meta.meta Integer . methods . sort . should_equal ['bit_shift_l', 'round', 'truncate']
+            Meta.meta Integer . methods . sort . should_equal ['%', '*', '+', '-', '/', '<', '<=', '>', '>=', '^', 'abs', 'bit_and', 'bit_not', 'bit_or', 'bit_shift', 'bit_shift_l', 'bit_shift_r', 'bit_xor', 'ceil', 'div', 'floor', 'negate', 'round', 'to_decimal', 'truncate']
```

Comment:
You change seems to partially address 
- #6303

Reply:
I am not sure about that - these methods are just a union of methods from `Small_Integer` and `Big_Integer`. There are no methods from `Any`.

Reply:
Right. Methods from `Any` are missing, but the behavior is much better!